hack/verify-bindata.sh

```diff
@@ -0,0 +1,11 @@
+set -euo pipefail
+
+echo "Verifying pkg/assets/bindata.go is up to date"
+
+TMP_OUTPUT="$(mktemp)"
+
+REPO_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )/.."
```

Comment:
Took me a while to figure out what this does. You can also `git rev-parse --show-toplevel` to get the root of the git repo.

Reply:
I like this because it doesn't require a git repo (as in a container build)